proto/stream_plan.proto

```diff
@@ -261,11 +261,24 @@ message GroupTopNNode {
   bool with_ties = 6;
 }
 
+message DeltaExpression {
+  expr.ExprNode.Type delta_type = 1;
+  expr.ExprNode delta = 2;
+}
+
+message InequalityPair {
```

Comment:
```suggestion
message BandJoinCondition {
```

Reply:
should we need to distinguish “GreaterEqual” and "greater" here?

Reply:
> 

fixed